airmozilla/manage/templates/manage/survey_edit.html

```diff
@@ -40,58 +29,24 @@
     {% include 'manage/_default_form.html' %}
 
     <h3>Questions</h3>
+    <a class="survey-links" href="{{ url('manage:survey_questions', survey.id) }}">Edit questions here</a>
```

Comment:
Also, instead of using CSS to make it a block element, wrap the `<a>` tag in something that *is* a block element like
```html
<p>
  <a class="survey-link" ...>
</p>
```